### ISSUE TITLE
Issue with response data while reading `scriptElement.src` after loading with Yett

### ISSUE THREAD

**Issue by dhruvppatel** [snipsco/yett#21]
Hello Team, Cheers to all the contributors for making this simple and powerful library. We have recently implemented this library to our website for GDPR compliance across the globe.

After integrating we have found one particular issue with Google IMA SDK Library. 

<img width="1340" alt="Screen Shot 2020-04-08 at 11 24 50 PM" src="https://user-images.githubusercontent.com/11784979/78817141-55fd5880-79f0-11ea-9620-4fbe544adb04.png">

After debugging I found that the IMA SDK script internally validates how the script has been loaded. Basically they iterate over all script elements present and read their `src` property and compare with actual script URL using Regex.

**With Yett Script**

<img width="1679" alt="Screen Shot 2020-04-08 at 11 30 20 PM" src="https://user-images.githubusercontent.com/11784979/78818039-a88b4480-79f1-11ea-8644-db308b7c6d21.png">

As you can see `script.src` returns `//imasdk.googleapis.com/js/sdkloader/ima3.js` which we have passed while loading.

**Without Yett Script**

<img width="1680" alt="Screen Shot 2020-04-08 at 11 30 08 PM" src="https://user-images.githubusercontent.com/11784979/78818061-af19bc00-79f1-11ea-842b-c8c86772ec2e.png">

As you can see `script.src` returns `https://imasdk.googleapis.com/js/sdkloader/ima3.js` with the protocol. 

It seems that `monkey.js` `src` `get` method is not working as expected. So for testing, I have forked the repo and modified to use `originalDescriptors` approach(which I found from your git history).

You can see the working branch code at https://github.com/dhruvppatel/yett/tree/fix/get-src

For testing, you can add the following snippet in doc `index.html` below `dynamic.js` script loading code and verify.

```
        (function() {
            var script = document.createElement('script')
            script.src = '//imasdk.googleapis.com/js/sdkloader/ima3.js'
            document.head.appendChild(script)
            setTimeout(function() {
                console.log(script.src);
            }, 1000);
        })();
```

**Comment by dhruvppatel** [comment]
@elbywan let me know your thoughts on this issue.

**Comment by elbywan** [comment]
Hi @dhruvppatel,

> @elbywan let me know your thoughts on this issue.

You're absolutely right, the behaviour from the `originalDescriptors` src getter is slightly different than just calling `script.src` for urls starting with `//`. Thanks a bunch for reporting this 👍 !

I restored the old way of doing things and published version `0.1.12` of the package with the fix 📦.

**Comment by dhruvppatel** [comment]
Superb!! Thanks for the update.
